iron-form.html

```diff
@@ -456,6 +457,20 @@
 
         this.fire('iron-form-reset');
       }, 1);
+    },
+
+    _isChildOfRegisteredParent: function(node) {
+      var parent = Polymer.dom(node).parentNode;
```

Comment:
So if `node = this._customElements[0]`, this method returns `false`
Is the desired behavior?